lib/internal/test_runner/reporter/tap.js

```diff
@@ -10,10 +10,12 @@ const {
   StringPrototypeReplaceAll,
   StringPrototypeSplit,
   StringPrototypeRepeat,
+  StringPrototypeStartsWith
 } = primordials;
 const { inspectWithNoCustomRetry } = require('internal/errors');
 const { isError, kEmptyObject } = require('internal/util');
 const { relative } = require('path');
+const util = require('util');
```

Comment:
We should use internal/util/inspect.js I think. Also use destructuring to reduce the chance of things being tampered with.